Dockerfile

```diff
@@ -1,4 +1,4 @@
 FROM alpine:latest
```

Comment:
Have you considered and discarded the idea of doing a multi-stage build and having the webhook container build `FROM scratch`?

Reply:
Not at all! Please submit a PR if you have any ideas